tests/5.1/target_teams_distribute_parallel_for/test_target_teams_distribute_parallel_for_default.c

```diff
@@ -0,0 +1,89 @@
+//-----------------test_target_teams_distribute_parallel_for_default.c-----
+//
+// OpenMP API Version 5.1 Mar 2023
+//
+// This test intends to test the functionality of an indirect
+// declare target region. A random int is chosen that correlates
+// to a function pointer. That function is then chosen and the
+// test ensures it runs as expected.
+// --------------------------------------------------------------
+
+
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+
+
+int DefaultFirstPrivate() {
+  int ErrCount = 0;
+  int Arr[32];
+  for (int i = 0; i < 32; ++i) {
+    Arr[i] = i;
+  }
+
+#pragma omp target teams distribute parallel for num_teams(2) thread_limit(10)\
+        default(firstprivate) map(tofrom: ErrCount)
+  for (int i = 0; i < 32; ++i) {
+    if (Arr[i] != i) {
+#pragma omp atomic
+      ErrCount += 1;
+    }
+  }
+  return ErrCount;
+}
+
+int DefaultPrivate() {
+  int ErrCount = 0;
+  int CONST = 123;
+  int Arr[32];
+  for (int i = 0; i < 32; ++i) {
+    Arr[i] = i;
+  }
+
+#pragma omp target teams distribute parallel for num_teams(2) thread_limit(10)\
+        default(private)
+  for (int i = 0; i < 32; ++i) {
+    CONST = 10;
+    Arr[i] += CONST;
```

Comment:
`Arr` should be explicitly mapped (e.g., `map(to: Arr)`).

Reply:
Hi @seyonglee ,
Thank you for pointing out, mapped the variables appropriately.
Please review the changes...